.circleci/python_ci.yml

```diff
@@ -0,0 +1,22 @@
+version: 2.1
+
+jobs:
+    install:
```

Comment:
```suggestion
    install_py3:
```

Reply:
I think ideally we could have two jobs: one w/ python 3 (this one that exists) and one w/ python 2 maybe called `install_py2`. It would mostly be identical to this one, but pulls the `svalinn/pymoab_py2-18.04` docker image.